contrib/openshift/osba-os-template.yaml

```diff
@@ -0,0 +1,238 @@
+          volumeMounts:
+          - name: redis-data
+            mountPath: /bitnami/redis
+        volumes:
+        - name: redis-data
+          persistentVolumeClaim:
+            claimName: osba-redis-pv-claim
+
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: osba-open-service-broker-azure
+    labels:
+      app: osba-open-service-broker-azure
+      release: "osba"
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: osba-open-service-broker-azure
+    template:
+      metadata:
+        labels:
+          app: osba-open-service-broker-azure
+          release: "osba"
+      spec:
+        containers:
+          - name: open-service-broker-azure
+            image: microsoft/azure-service-broker:v0.10.0
+            imagePullPolicy: IfNotPresent
+            env:
+            - name: ENVIRONMENT
+              value: ${ENVIRONMENT}
+            - name: AZURE_SUBSCRIPTION_ID
+              value: ${AZURE_SUBSCRIPTION_ID}
+            - name: AZURE_TENANT_ID
+              value: ${AZURE_TENANT_ID}
+            - name: AZURE_CLIENT_ID
+              value: ${AZURE_CLIENT_ID}
+            - name: AZURE_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: azure-client-secret
+            - name: AZURE_DEFAULT_LOCATION
+              value: ${AZURE_DEFAULT_LOCATION}
+            - name: REDIS_HOST
+              value: osba-redis
+            - name: REDIS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: redis-password
+            - name: AES256_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: encryption-key
+            - name: BASIC_AUTH_USERNAME
+              value: username
+            - name: BASIC_AUTH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: basic-auth-password
+            - name: MIN_STABILITY
+              value: EXPERIMENTAL
+            ports:
+            - containerPort: 8080
+            readinessProbe:
+              tcpSocket:
+                port: 8080
+              failureThreshold: 1
+              initialDelaySeconds: 10
+              periodSeconds: 10
+              successThreshold: 1
+              timeoutSeconds: 2
+            livenessProbe:
+              tcpSocket:
+                port: 8080
+              failureThreshold: 3
+              initialDelaySeconds: 30
+              periodSeconds: 10
+              successThreshold: 1
+              timeoutSeconds: 2
+- apiVersion: servicecatalog.k8s.io/v1beta1
+  kind: ClusterServiceBroker
+  metadata:
+    name: osba
+  spec:
+    url: http://osba-open-service-broker-azure.osba.svc.cluster.local
```

Comment:
This looks like it assumes the broker has been deployed in the `osba` namespace. That isn't a good assumption.